test/node/src/schema.test.ts

```diff
@@ -638,6 +753,65 @@ for (const dialect of DIALECTS) {
         await builder.execute()
       })
 
+      if (dialect !== 'mysql' && dialect !== 'mssql') {
```

Comment:
nitpicky

more explicit.

```suggestion
      if (dialect === 'postgres' || dialect === 'sqlite') {
```

Reply:
Not really. Mine will include all new dialects. Yours will exclude them.